sdk/search/search-documents/CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 ## 11.1.0 (2021-02-11)
 
 - GA Release for `@azure/search-documents`.
-- Linter Errors in the SDK code have been fixed. Please refer [#13060](https://github.com/Azure/azure-sdk-for-js/pull/13060) for further details.
+
+### Changes since 11.0.3
+
+- [Breaking] The `@azure/search-documents` has been moved to V6 SDK Generator. Please refer [#12829](https://github.com/Azure/azure-sdk-for-js/pull/12829) for further details.
```

Comment:
Why is this breaking? Additionally, we should have the changelog update reflecting how this affects the end user. The only thing I can think of is the enum related stuff that would have changed. I would mention that rather than the use of different code generator

Reply:
It was because of the enum values. Let me try to rephrase it.

Reply:
Rephrased the lines and also noted the one rename.